### DISCUSSION TITLE
In Doris v1.2.4, an error occurred when attempting to read from another Doris cluster using catalog

### DISCUSSION THREAD

**Discussion by daydayupon** [apache/doris#29824] [A - General / Q&A]
现在源doris的版本是1.2.4，然后创建了catalog读取外部的doris数据库集群，同步schema的时候没有问题，但是查询的时候一直报如下的错误，请问是哪里出现了问题，catalog中使用的jdbc版本5.1.x和8.0.x都试过了，都会有同样的问题。
![Uploading WechatIMG1734.jpg…]()

**Comment by LemonLiTree** [comment]
看不到你的图，可以升级2.0 版本看看 ，但并不推荐这么做。

**Reply by daydayupon** [reply]
那在1.2.4版本下，对于从另一个doris同步数据有什么推荐的方式吗

**Reply by LemonLiTree** [reply]
backup restore
